server/routes/selectAssociatedStaff/selectAssociatedStaff.test.ts

```diff
@@ -73,11 +92,11 @@ describe('GET /select-associated-staff', () => {
 describe('POST /select-associated-staff', () => {
   it('should redirect to select staff member page with the correct search text and redirect URL intact', () => {
     return request(app)
-      .post('/select-associated-staff?searchFirstName=john&searchLastName=smith')
-      .send({ searchFirstName: 'john', searchLastName: 'doe' })
+      .post('/select-associated-staff?staffFirstName=john&staffLastName=smith')
```

Comment:
A bit confused by this (sorry, didn't spot this last time) - how come the first/last name are in the URL and the post body?